source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -318,32 +440,73 @@ def get_matrix_params(self, origins, destinations, mode, max_duration, request):
         datetime = request['datetime']
         params['departure'] = _str_to_dt(datetime)
 
-        # Note: for the moment we limit to self.max_points the number of n:m asked
-        for i, o in enumerate(origins[: self.max_points]):
+        for i, o in enumerate(origins[: int(max_matrix_points)]):
```

Comment:
just a little python tip, the operator `[]` in python is a simple copy, it will make allocations.. you can use `islice`... for the sake for performance...  
```python
import itertools
for i, o in enumerate(itertools.islice(origins, max_matrix_points)):
```
in this case, since the origins and destinations are of small size, it won't change anything...

Reply:
**origins** or **destinatons** are smaller than 100, but I like the tip.
thanks :1st_place_medal: